src/FixedDataTableBufferedRows.react.js

```diff
@@ -117,12 +117,22 @@ var FixedDataTableBufferedRows = React.createClass({
     var rowPositionGetter = props.rowPositionGetter;
 
     var rowsToRender = this.state.rowsToRender;
+
+    //Sort the rows, we slice first to avoid changing original
+    var sortedRowsToRender = rowsToRender.slice().sort((a, b) => a - b);
+    var rowPositions = {};
+
+    //Row position calculation requires that rows are calculated in order
+    sortedRowsToRender.forEach((rowIndex) => {
+      rowPositions[rowIndex] = rowPositionGetter(rowIndex);
```

Comment:
No nLogN performance issue, I misread the code somewhere. 

If m is the visible row count, we have m log m from the sort, and m log n from rowPositionGetter. The total complexity remains unchanged, since m > n.

I agree that we could manually iterate on the first position getter, reducing it to m + logN, but I think we should do that in a follow up review. I'm not imagining the priority for that is super high, plus its out of scope for this bug.